### PR TITLE
Refactor smartFAQ queries

### LINKED ISSUE #184: Refactor smartFAQ queries
- do not make a request to `/api/v1/categories` (without ID) - just always assume category with ID `3` is in what we are interested in and return categories for this ID (currently categories under id `1` are being fetched)
- use `tree_ids=3` in REST API search requests to search only in smartfaq FAQs

### PR DESCRIPTION
**Pull request by radkag**
Closes #184.

### CODE REVIEW

File: config/application.js [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 import url from 'url';
 
+const FAQ_CATEGORY_ID = '3';
```
Comment (mrtnzlml) [inline config/application.js]: Please add a small comment explaining this magic number. Thank you.

**Comment by mrtnzlml** [comment]
@radkag Tests are not happy with this change... :)

**Comment by jaroslav-kubicek** [comment]
rebased, ready for final CR

File: config/application.js [hunk 2]
```diff
@@ -30,13 +36,10 @@ export default {
...
-        { q: search, autocomplete: true },
+        { q: search, autocomplete: true, tree_ids: FAQ_CATEGORY_ID },
       ),
-    allFAQCategories: (faqCategoryId: ?number) => {
-      const categoryId = faqCategoryId ? `/${faqCategoryId}` : '';
-
-      return `https://api.skypicker.com/knowledgebase/api/v1/categories${categoryId}`;
-    },
+    allFAQCategories: () =>
+      `https://api.skypicker.com/knowledgebase/api/v1/categories/${FAQ_CATEGORY_ID}`,
```
Comment (mrtnzlml) [inline config/application.js]: This could be moved to the dataloaders as we now know... :)

Comment (jaroslav-kubicek) [reply config/application.js]: fixed maybe?

**Comment by jaroslav-kubicek** [comment]
> Looks good from my point of view. @jaroslav-kubicek please review it as well and merge. I don't know the context here well.

Ouch, I meant rebased, _it's ok from my side_ and ready for final CR, forgot to do it as approval  🙃

**Pull request merged**